Tests/Integration/SPM/project.yml

```diff
@@ -4,7 +4,7 @@ options:
 packages:
   MapboxCommon:
     url: git@github.com:mapbox/mapbox-common-ios.git
-    branch: release-v24.0.0-rc.2
+    branch: v24.9.0-daily-2024-11-12-04-00
```

Comment:
Created a dummy branch with the fix to turfVersion so the test may pass
Otherwise with 24.9.0-daily-2024-11-12-04-26 it will fail
It will be overwritten with proper version with first daily release